File: lotto_api.py
import requests

url = 'https://www.dhlottery.co.kr/common.do?method=getLottoNumber&drwNo=1'

res = requests.get(url)

#json을 python dictionary로 파싱
data = res.json()

winner = []

# js의 push와 거의 동읠
winner.append(data['drwtNo1'])
winner.append(data['drwtNo2'])
winner.append(data['drwtNo3'])
winner.append(data['drwtNo4'])
winner.append(data['drwtNo5'])
winner.append(data['drwtNo6'])

File: lotto_api.js
/*
axios를 활용하여
https://www.dhlottery.co.kr/common.do?method=getLottoNumber&drwNo=1에 요청을 보내
1등 번호 6개가 담긴 winner 배열을 만들어 출력하세요.
*/

import axios from 'axios';
    componentDidMount = () => {
        axios.get (
            'https://www.dhlottery.co.kr/common.do?method=getLottoNumber&drwNo=1'
        )
        .then((response) => {
            let winner = []
        for (i=0, i<=6, i++) {
            
        }
            console.log()
        }
    )
    
}